site/src/_pages/spec/1_block-types.mdx

```diff
@@ -25,10 +34,10 @@ A block package SHOULD contain:
 
     - a `name` for the block
     - any libraries the block expects the embedding application to supply it with under `externals` – i.e. libraries which the block depends on but does not include in its package – using the name of the library as it is usually distributed (e.g. via npm), and the expected version (or version range).
-      For example, a block may rely on React (`"externals": [{ "react": "^16.0.0" }]`), but assume that the embedding application will provide it, to save loading it multiple times on a page. This is also a useful way of indicating how the block expects to be rendered.
+      For example, a block may rely on React (`"externals": [{ "react": "^16.0.0" }]`), but assume that the embedding application will provide it, to save loading it multiple times on a page.
     - the path or URL to the entrypoint source file (e.g. `index.html`, `index.js`) under `source`
     - the `version` of the block, which SHOULD use [semantic versioning](https://semver.org/)
-    - `protocol`: the applicable block protocol version
+    - `protocol`: the applicable block protocol version, once the protocol is versioned (it is currently not).
```

Comment:
@CiaranMn Do we want to do an 0.1 version and adopt this? Happy to continue this conversation on the dev Slack.

Reply:
Yes - let's make it `v0.1` as of Thursday (with any other changes brought in)